src/user/user.service.ts

```diff
@@ -1,4 +1,35 @@
-import { Injectable } from "@nestjs/common";
+import { Injectable, Inject } from "@nestjs/common";
+import { User } from "../data/entities/user.entity";
```

Comment:
Nitpick: unrelated but an improvement to remove relative paths (aka the devil): https://medium.com/@fmoessle/typescript-paths-with-ts-node-ts-node-dev-and-jest-671deacf6428